src/include/common/file_utils.h

```diff
@@ -63,6 +63,12 @@ class FileUtils {
         return std::filesystem::exists(path);
     }
 
+    static inline void createDirIfNotExists(const std::string& path) {
```

Comment:
Better to move this function close to `static void createDir(const std::string& dir);`.